examples/java/spring-boot/src/test/java/uk/gov/api/springboot/models/metadata/BulkMetadataResponseTest.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.api.springboot.models.metadata;
+package uk.gov.api.springboot.models.metadata.v1alpha;
```

Comment:
do we need to change the actual package these tests are in (at the moment they're still in `uk/gov/api/springboot/models/metadata/`)?

Reply:
Great spot! Updated both the `package` and the directory hierarchy :+1: